core/src/CoreTelemetryService.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Core
+{
+    internal class CoreTelemetryService
```

Comment:
mark class as static

Reply:
not what I meant. Everywhere else we have a mathod called GetInstance. Personally I prefer the property called Instance. Most of our similar code follows the pattern of a method named GetInstance() ... thus property = Intance, or method = GetInstance()

---
In reply to: [208828711](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1173#discussion_r208828711) [](ancestors = 208828711)